.github/workflows/cla-check.yaml

```diff
@@ -7,6 +7,4 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Check if CLA signed
-        uses: canonical/has-signed-canonical-cla@v1
-        with:
-          accept-existing-contributors: true
+        uses: canonical/has-signed-canonical-cla@d8942819445fb93015c6f298e1636a23f6f47064
```

Comment:
Would you add a reference to https://github.com/canonical/has-signed-canonical-cla/issues/73 here?

Reply:
I had originally pinned it because I didn't know when we could expect a new release, but since opening the PR the `v2` tag has been updated to include the fix so I've dropped the commit pinning to the specific commit.